src/vm/gdbjit.cpp

```diff
@@ -1603,7 +1603,11 @@ void NotifyGdb::MethodCompiled(MethodDesc* MethodDescPtr)
     for (int i = 1 + method.GetCount(); i < SymbolCount; i++)
     {
         ++pShdr;
+#ifdef _TARGET_ARM_
+        pShdr->sh_addr = SymbolNames[i].m_value & ~1; // for THUMB code
```

Comment:
Can you please use PCODEToPINSTR macro instead of this `#ifdef` / `#else` / `#endif`?
And if you don't mind, could you please do the same in NotifyGdb::MethodCompiled?
Also, we set the THUMB bit in NotifyGdb::BuildSymbolTableSection. It would be great to use DataPointerToThumbCode there.